### PR TITLE
fix: remove struct and add TODO

### PR DESCRIPTION
**Pull request by jaimem88**
Fixes https://github.com/exercism/go/issues/1590

### CODE REVIEW

**Comment by jaimem88** [comment]
@iHiD @ErikSchierboom ready for review 😄

**Review by junedev — CHANGES_REQUESTED** [review]
Now that the stub is changed, the instructions need to be changed as well so that the student knows what fields to include in the struct. Currently e.g. distance is not mentioned at all (and it looks like even the example output in task 1 is incorrect).

**Comment by ErikSchierboom** [comment]
@jaimem88 I'll let @junedev review, she actually knows Go and I don't :)

**Comment by jaimem88** [comment]
Thank you both! I'll get back to this in a couple of days 🙇

File: exercises/concept/need-for-speed/.docs/instructions.md [hunk 1]
```diff
@@ -1,22 +1,32 @@
 # Instructions
 
-In this exercise you'll be organizing races between various types of remote controlled cars. Each car has its own speed and battery drain characteristics.
+In this exercise you'll be organizing races between various types of remote controlled cars.
+Each car has its own speed and battery drain characteristics.
```
Comment (jaimem88) [inline exercises/concept/need-for-speed/.docs/instructions.md:4]: not sure if there are specific guidelines for text, but it helps to read by splitting into multiple lines. I can revert if prefered :)

File: exercises/concept/need-for-speed/need_for_speed.go [hunk 2]
```diff
@@ -1,27 +1,16 @@
...
 	panic("Please implement the NewCar function")
 }
 
-
-// Track implements a race track.
-type Track struct{
-	distance int
-}
-
+// TODO: define the 'Track' type struct
```
Comment (jaimem88) [inline exercises/concept/need-for-speed/need_for_speed.go:10]: Removed the `type Track struct` part, it's also defined in the instructions 😄

File: exercises/concept/need-for-speed/need_for_speed.go [hunk 3]
```diff
@@ -1,27 +1,16 @@
...
 
 // NewTrack created a new track
-func NewTrack(distance int) Track{
+func NewTrack(distance int) Track {
```
Comment (jaimem88) [inline exercises/concept/need-for-speed/need_for_speed.go:13]: I run goimports locally so it took care of this

**Comment by jaimem88** [comment]
> Thanks for taking care of this! Now that the stub is changed, the instructions need to be changed as well so that the student knows what fields to include in the struct. Currently e.g. distance is not mentioned at all (and it looks like even the example output in task 1 is incorrect). Would be great if you could include this in your PR so the content remains consistent.

@junedev I've updated the docs a bit, let me know what you think!

**Review by junedev — APPROVED** [review]

File: exercises/concept/need-for-speed/.docs/instructions.md (hunk 1 above)
Comment (ErikSchierboom) [reply exercises/concept/need-for-speed/.docs/instructions.md:4]: This is totally fine!

**Comment by ErikSchierboom** [comment]
Thanks again!

**Pull request merged**